Mathlib/MeasureTheory/Constructions/UnitInterval.lean

```diff
@@ -0,0 +1,38 @@
+/-
+Copyright (c) 2024 Eric Wieser. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Eric Wieser
+-/
+import Mathlib.Topology.UnitInterval
+import Mathlib.MeasureTheory.Constructions.BorelSpace.Basic
+import Mathlib.MeasureTheory.Measure.Haar.OfBasis
+import Mathlib.MeasureTheory.Measure.Lebesgue.Basic
+
+/-!
+# The canonical measure on the unit interval
+
+This file provides a `MeasureTheory.MeasureSpace` instance on `unitInterval`,
+and shows it is a probability measure.
+-/
+open scoped unitInterval
+open MeasureTheory
+
+namespace unitInterval
+
+noncomputable instance : MeasureSpace I := Measure.Subtype.measureSpace
+
+theorem volume_def : (volume : Measure I) = volume.comap Subtype.val := rfl
+
+instance : IsProbabilityMeasure (volume : Measure I) where
+  measure_univ := by
+    rw [Measure.Subtype.volume_univ measurableSet_Icc.nullMeasurableSet, Real.volume_Icc, sub_zero,
+      ENNReal.ofReal_one]
+
+@[measurability]
+theorem measurable_symm : Measurable symm := continuous_symm.measurable
+
+/-- `unitInterval.symm` as a `MeasurableEquiv`. -/
+@[simps]
+def symmMeasurableEquiv : I ≃ᵐ I := symmHomeomorph.toMeasurableEquiv
```

Comment:
@urkud, do you think this is worth having, or should I delete it?

Reply:
I would remove it as it's so easy to access it from the homeomorphism.

Reply:
Done